docs/pages/product/apis-integrations/_meta.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   "sql-api": "SQL API",
   "rest-api": "REST API",
   "graphql-api": "GraphQL API",
+  "mdx-api": "MDX API",
```

Comment:
Following [this discussion](https://cubedevinc.slack.com/archives/C019TT9S57W/p1717414449720169), it would be great to also include the MDX API to the API overview page.